modules/ROOT/pages/roles.adoc

```diff
@@ -265,10 +271,70 @@ Custom roles have a radial button next to them.
 
 You can set up your Anypoint Platform organization so that when a SAML user belongs to certain groups, Anypoint Platform automatically grants certain equivalent roles in your Anypoint Platform organization.
 
+== Migrate Roles to Teams
+
+After you opt in to the Teams feature, you can convert existing roles into permissions, convert roles to teams, or merge roles with teams.
+
+When you convert or merge a role, the role is deleted.
+
+=== Convert a Role to Permissions
+
+When you convert roles to permissions, users keep the same permissions that they previously received with roles.
+
+[WARNING]
+When you convert a role to permissions, the users that belonged to the role are no longer grouped according to their permissions and must be managed individually. Additionally, roles that can now be applied as (or converted to) permissions, such as the Exchange Contributors permission, can be managed only while you are using the Teams feature. 
```

Comment:
Did you mean for this sentence to begin exactly like the one before it, or did you mean to delete/incorporate the one before it?

Reply:
Does this mean the same thing?
```suggestion
When you convert a role to permissions, the users that belonged to the role are no longer grouped according to their permissions and must be managed individually. Additionally, roles that are now permissions, such as the Exchange Contributors permission, are managed only while you are using the Teams feature. 
```

Reply:
The distinction is important here since this is a change made with the Teams update.